tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -26,9 +24,6 @@
     <PackageReference Include="log4net.Ext.Json" Version="2.0.10.1" Condition="'$(TargetFramework)' == 'net48'" />
 
     <!-- log4net .NET core references -->
-    <PackageReference Include="log4net" Version="2.0.6 " Condition="'$(TargetFramework)' == 'netcoreapp2.1'" />
```

Comment:
I believe log4net 2.0.6 was the first .NET Core version, still? (probably without the JSON formatting)

Reply:
This comment is not specific to your comment above but it is related. There is currently a general limitation of how we test different version of a library. We can only test different versions of a library if they are referenced by different frameworks. We don't have the ability to test different versions of a library running on a specific framework. Given that, we have a limited amount of setup to test the versions of the library we want to test.

Come back to your question, i feel it is ok to drop testing for 2.0.6 because it is the version that has security vulnerability.